src/lib/walletgen.js

```diff
@@ -44,6 +45,27 @@ export const makeTicket = point => {
   });
 };
 
+export const makeDeterministicTicket = (point, seed) => {
+  const bits =
+    point < MIN_STAR
+      ? GALAXY_ENTROPY_BITS
+      : point < MIN_PLANET
+      ? STAR_ENTROPY_BITS
+      : PLANET_ENTROPY_BITS;
```

Comment:
This feels like it should be its own utility function. It's used in `makeTicket` also.